#### açıklama

Bu örnektekiana fikri anlamak biraz zor olabilir. <br>

Elimizde bir sınıf hiyerarşisi içinde bulunan bir _exception_ sınıfı türünden bir hata nesnesi var.
Diikat: burada söz konusu hata sınıfı hiyerarşisi içinde aynı taban sınıftan kalıtm yoluyla elde edilmiş çok sayıda özelleştirilmiş _exception_ sınıfı olabilir
Bu hata nesnesini bir fonksiyona göndermek ve fonksiyonun dinamik türü koruyarak gönderdiğimiz hata nesnesini _throw_ etmesini (göndermesini) istiyoruz.

<!--

-->
